src/main/app/drafts/models/draftLegalClaim.ts

```diff
@@ -28,6 +28,11 @@ export class DraftLegalClaim extends DraftDocument implements Serializable<Draft
   feeAccount: FeeAccount = new FeeAccount()
   feeAmountInPennies: number
   feeCode: string
+  // View Draft items
+  isDefendantDeleted: boolean = false
```

Comment:
I don't think these toggles belong at the top level of this draft document.

Reply:
I'm guessing you're suggesting I put them inside other form models?